frontend/src/components/pages/CreateReview/AddMultiSelect.tsx

```diff
@@ -58,12 +100,35 @@ const AddMultiSelect = ({
   allowDeleteOption,
   allowAddOption,
   allowMultiSelectOption,
+  searchStyle,
 }: AddMultiSelectProps): React.ReactElement => {
   const [showModal, setShowModal] = useState<boolean>(false);
   const [optionToDelete, setOptionToDelete] = useState<Option>(EmptyOption);
 
   const onClose = () => setShowModal(false);
 
+  customSearchStyles.control = useMemo(
+    () =>
+      optionsSelected.length > 0
+        ? (provided: any) => ({
+            ...provided,
+            color: "gray.900",
```

Comment:
Really clever code here, but maybe pull the styles out into a constant variable at the top of the file just for readability, i.e:

```
EMPTY_FILTER_STYLES = {color: "gray.900", ...}
NON_EMPTY_FILTER_STYLES = {"color": "gray.900", ...}
```

then just call it here like:
```
(provided: any) => ({...provided, ...EMPTY_FILTER_STYLES}) 
```